### ISSUE TITLE
Case Sensitivity Issues

### ISSUE THREAD

**Issue by bethSQL** [gwalkey/SQLTranscriptase#10]
There are a number of errors when run against a case-sensitive server. Here are the ones that we found in testing:

01 - Server Settings
"Database 'Master' does not exist. Make sure that the name is entered correctly." [line 175]

01 - Server DataBases
The multi-part identifier "m.type_desc" could not be bound.
The multi-part identifier "m.type_desc" could not be bound.
The multi-part identifier "m.database_id" could not be bound.
The multi-part identifier "d.create_date" could not be bound.
... more ...
Make case consistent lines 126-189 with the table aliases d and m

04 - Agent Alerts
Invalid column name 'NAME'. [lines 92-130, 4 places]

04 - Agent Schedules
Invalid column name 'Schedule_uid'. [line 330]
Invalid column name 'Freq_SubDay_Interval'. [378]
Invalid column name 'starthour'." [770]
Invalid column name 'job_enabled'. [784]
Invalid column name 'job'." [785]

34 - User Objects in Master DB
Invalid object name 'Master.sys.objects'." [fix case in lines 119-122]

50 - Security Tree
The multi-part identifier "sprinc.name" could not be bound." [line 272]

**Comment by gwalkey** [comment]
01 - Server Settings
"Database 'Master' does not exist. Make sure that the name is entered correctly." [line 175]
is that on a SQL 7 instance? Works fine on 30+ servers from v2005 to v2019

01 - Server DataBases
No version of SQL server throws this error
"Make case consistent lines 126-189 with the table aliases d and m
You are free to do that yourself

I have a feeling you are using PS 6 or 7, which these are not guaranteed to work on, 
as PS 6/7 is not compatible with 5.1

04 - Agent Alerts
what version of SQL are you on?
must be a sysadmin to run these
No error found

04 - Agent Schedules
No Error Found

04 - Agent Schedules
No Errors  found

34 - User Objects in Master DB
Invalid object name 'Master.sys.objects'." [fix case in lines 119-122]
Need sysadmin to run this
No Errors Found

50 - Security Tree
The multi-part identifier "sprinc.name" could not be bound." [line 272]
Need to be sysadmin to run

thanks for the tips

**Comment by bethSQL** [comment]
I understand what you are saying. As you have created this as a great resource to the community, I want it to serve all situations, especially for users who are not PS coders. 

If you run this on a case sensitive (CS) server, you will see these issues. Most of our 70 servers are CI, but about 15 are CS, which is not the norm. So lower case master works in CS, where mixed-case Master does not. 

I have made the fixes in my download, which is why I provided the line numbers so the base code can be robust. Are you willing to let me directly update your project? I'm not knowledgeable about that, but I can learn.
